### DISCUSSION TITLE
Draft issue doesn't send Notifications.

### DISCUSSION THREAD

**Discussion by mamtoraah** [community/community#119825] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

I have recently started using github projects, I added my friend to the private github project. @Harsh-31 

I believe github projects would eventually replace JIRA atlassian and similar stuff. 

One big issue. Neither me nor my friend is getting any notification when issues are assigned.

It says Draft issue. And there is an option to convert to Issue and choose a repo.

But I want notifcation to be sent if it is assigned already to my friend. At least email notification.

Please help

<img width="1396" alt="image" src="https://github.com/community/community/assets/32389544/de80caad-8edb-4de1-a885-ca4e9e945a21">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by DorisFangWork** [comment]
Same concerns here, I would like to use Project to manage my task assignment. However, there is no notification function. Not every task is a issue, or link to any Repo code base, some tasks are just documentations or discussion related.

**Comment by mrybak123** [comment]
Never assigned Draft Issues shouldn't be called Drafts. You could keep the Draft concept and behave as you do (not sending notifications out) in that state, but then allow to convert the Draft into a "real" issue without assigning it to a particular repo, but perhaps the organization instead. Then send out the notification.
